html/js/main.js

```diff
@@ -0,0 +1,23 @@
+(function () {
+    "use strict";
+    // Get global canvas and store it in GAMBIT
+    // GAMBIT is initialized by the library
+    GAMBIT.canvas = SVG('canvas').size("100%", "100%").attr({'style': 'background: #fff'});
+    GAMBIT.tools = new GAMBIT.UI.Tools();
+
+    // // Create a tree and draw it
```

Comment:
Consider (I am not sure about this) adding here listeners to all the toolbar buttons, instead of writing them inline in HTML